catalog-model/src/main/java/org/openecomp/sdc/be/model/jsonjanusgraph/operations/ToscaElementOperation.java

```diff
@@ -1379,10 +1379,7 @@ private boolean isAddToCatalog(List<ResourceTypeEnum> excludeTypes, Map<String,
         if (resourceTypeStr != null) {
             ResourceTypeEnum resourceType = ResourceTypeEnum.getType((String) resourceTypeStr);
             if (!CollectionUtils.isEmpty(excludeTypes)) {
-                Optional<ResourceTypeEnum> op = excludeTypes.stream().filter(rt -> rt == resourceType).findAny();
-                if (op.isPresent()) {
-                    isAddToCatalog = false;
-                }
+                isAddToCatalog = not (excludeTypes.stream().anyMatch(rt -> rt == resourceType);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;)&#34; to complete ArgumentList

Reply:
damn